doc/man3/ERR_clear_last_constant_time.pod

```diff
@@ -0,0 +1,31 @@
+=pod
+
+=head1 NAME
+
+ERR_clear_last_constant_time - clear last error in constant time
+
+=head1 SYNOPSIS
+
+ #include <openssl/err.h>
+
+ void ERR_clear_last_constant_time(int clear);
+
+=head1 DESCRIPTION
+
+ERR_clear_last_constant_time() removes the last error raised by L<ERR_raise(3)>
+from the error stack in constant time.
+
```

Comment:
We should probably describe the possible values of clear. The code in theory supports 0 and something other than 0, but all internal callers use 0 and 1, and it's fine to document it as such.

It's probably also useful to indicate that you should unconditionally add the error and then call this function to remove it in case there is no error.